speakers.py

```diff
@@ -0,0 +1,321 @@
+        'I am aware of the requirement to participate in dry runs, and will do my best to make myself available': 'dry_run',
+    }
+
+    answers = {}
+    for a in speaker['answers']:
+        try:
+            answers[QA_MAP[a['question']['question']['en']]] = a['answer']
+        except KeyError:
+            answers[QA_MAP[a['question']['question']['he']]] = a['answer']
+    return answers
+
+
+def get_data(endpoint, token, verbose=False):
+    url = endpoint + "?" + urllib.parse.urlencode({'limit': 10000, 'questions': 'all'})
+
+    if verbose:
+        print(f"Fetching {url}")
+
+    headers = {'Authorization': f'token {token}'}
+    response = requests.get(url, headers=headers)
+
+    if response.status_code != 200:
+        print(f"Error fetching data from '{endpoint}' code '{response.status_code}'")
+        return {}
+    elif verbose:
+        print("Success")
+
+    speakers = []
+    for speaker in response.json()['results']:
+        speaker['answers'] = get_speaker_answers(speaker)
+        speakers.append(speaker)
+
+    if verbose:
+        print(f"Count: {response.json().get('count', 0)}")
+        print(f"Next: {response.json().get('next', 0)}")
+        print(f"Previous: {response.json().get('previous', 0)}")
+
+        stats = {}
+        for speaker in speakers:
+            for a in speaker['answers']:
+                stats[a] = stats.get(a, 0) + 1
+        pprint(stats)
+
+    return speakers
+
+
+def fetch_handler(event, item_type, token, verbose=False):
+
+    if item_type == 'speakers':
+        url = SPEAKERS_URL_PATH.format(event=event)
+    elif item_type == 'talks':
+        url = TALKS_URL_PATH.format(event=event)
+    else:
+        raise ValueError(f"Unknown item type: {item_type}")
+
+    print(f"Fetching data from '{DOMAIN}{url}'...")
+    data = get_data(f"{DOMAIN}{url}", token, verbose=verbose)
+    return data
+
+
+def filter_speakers(speakers, talks):
+
+    accepted_talks = [t['code'] for t in talks if t['state'] == 'confirmed']
+    # accepted_speakers = set(
+    #     s['code'] for t in talks for s in t['speakers'] if t['state'] == 'confirmed'
+    # )
+
+    # accepted = []
+    # for speaker in speakers:
+    #     if speaker['code'] in accepted_speakers:
+    #         accepted.append(speaker)
+
+    accepted = []
+    for speaker in speakers:
+        for talk in speaker['submissions']:
+            if talk in accepted_talks and speaker['answers'].get('agree_to_publish', 'No') == 'True':
+                accepted.append(speaker)
+                # break
+
+    print('-----')
+    print(f"Accepted talks: {len(accepted_talks)}")
+    # print(f"Accepted talks: {accepted_talks}")
+    print('-----')
+    print(f"Accepted speakers: {len(accepted)}")
+    # print(f"Accepted speakers: {accepted}")
+
+    return accepted
+
+
+SPAKERS_MD = """Title: Speakers
+Slug: speakers
+Template: speakers
+save_as: speakers.html
+Lang: en
+URL:
+page_number: 7
```

Comment:
I'm not sure what the empty URL setting is doing, and I think the speakers page should be after the agenda
(which 15).
```suggestion
page_number: 17
```
At some point, we should do a "renumber" to make the numbers 10 apart again